libraries/botbuilder-ai/botbuilder/ai/luis/luis_recognizer.py

```diff
@@ -362,3 +370,11 @@ def fill_luis_event_properties(
         )
 
         await turn_context.send_activity(trace_activity)
+
+    def _merge_options(
+        self,
+        userDefinedOptions
```

Comment:
This might be nitpicking so feel free to ignore it. 
Although this method is "private" I would consider to annotate it in the form:
`user_defined_options: LuisPredictionOptions `

Reply:
Not at all...great feedback : ) Ok, this i updated.